worker/sandbox/settings.go

```diff
@@ -0,0 +1,24 @@
+package sandbox
+
+type Settings struct {
+	LogSandbox bool
+	IgnoreWarning bool
+}
+
+var DefaultSettings = Settings{LogSandbox: true, IgnoreWarning: false}
```

Comment:
This should not be public, as public variables can be modified from outside.

Reply:
Sorry for the (very) late reply. Fixed this by adding a new function to parse options and construct a Settings.